experiments/react-native-uifabric/src/components/RadioGroup/RadioButton.tsx

```diff
@@ -0,0 +1,140 @@
+    // 2) Selects the currently focused button by calling the RadioGroup's callback function.
+    const onFocusChange = (ev: NativeSyntheticEvent<{}>) => {
+      pressable.props.onFocus && pressable.props.onFocus(ev);
+      // This check is necessary because this func gets called even when a button loses focus (not sure why?) which then calls the client's onChange multiple times
+      if (!state.selected) info.onButtonSelect(buttonKey);
+    };
+
+    const accessibilityTraits = {
+      accessibilityRole: 'radio',
+      accessibilityLabel: ariaLabel ? ariaLabel : content,
+      // Why are we able to do this? The radio button that's focused is always going to be the one selected. Therefore, unless it's disabled,
+      // we can always set state=selected for the narrator, because every time the narrator is focused on a button, that's the one selected
+      accessibilityStates: state.disabled ? ['disabled'] : ['selected'],
+      accessibilityActions: [{ name: 'Select', label: 'Select a RadioButton' }],
+      onAccessibilityAction: (event: { nativeEvent: { actionName: any } }) => {
+        switch (event.nativeEvent.actionName) {
+          case 'Select':
+            info.onButtonSelect(buttonKey);
+        }
+      }
+    };
+
+    //When the option is disabled, there's no built-in disable functionality for React-Native. So here's the logic I implemented for it.
+    const rootProps = state.disabled
+      ? { rest, ...accessibilityTraits }
+      : {
+          ...pressable.props,
+          ...accessibilityTraits,
+          rest,
+          onFocus: onFocusChange
+        };
+    const buttonProps = state.disabled ? { rest, style: { borderColor: 'grey' } } : rest;
+
+    let contentProps = {};
+    if (state.disabled) {
+      contentProps = { children: content, style: { color: 'grey' } };
+    } else if (state.focused) {
+      contentProps = { children: content, style: focusedStyle };
+    } else {
+      contentProps = { children: content };
+    }
+
+    // This handles the hovered/pressed UI functionality
+    let innerCircleProps = {};
+    if (state.selected) {
+      innerCircleProps = selectedStyle;
+    } else if (state.hovered || state.focused) {
+      innerCircleProps = hoveredStyle;
+    } else {
+      innerCircleProps = rest;
+    }
+
+    const slotProps = mergeSettings<IRadioButtonSlotProps>(styleProps, {
+      root: rootProps,
+      button: buttonProps,
+      innerCircle: innerCircleProps,
+      content: contentProps
+    });
+
+    return { slotProps, state };
+  },
+
+  render: (Slots: ISlots<IRadioButtonSlotProps>, renderData: IRadioButtonRenderData, ...children: React.ReactNode[]) => {
+    return (
+      <RadioGroupContext.Consumer>
+        {context => (
+          <Slots.root>
+            <Slots.button>
+              <Slots.innerCircle />
+            </Slots.button>
+            <Slots.content />
+            {children}
```

Comment:
do we actually want to put `children` here?

Reply:
For now, yes. In V2 I am going to change it up a bit to allow for Pickers, etc. to be children